.chocolatey/tools/ChocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,11 @@
+$InstallDir = Join-Path $env:APPDATA 'PresenceLight'
+$SettingsPath = Join-Path $InstallDir 'appsettings.json'

```

Comment:
Based on the testing that I did, the custom information was being added to a `settings.json` file, not the `appsettings.json` file.